src/content/app/entity-viewer/hooks/useEntityViewerAnalytics.ts

```diff
@@ -121,7 +123,7 @@ const useEntityViewerAnalytics = () => {
     trackExternalLinkClick('gene_view_transcript_list', label);
   };
 
-  const trackPreviouslyViewedLinkClick = (params: {
+  const trackBookmarksLinkClickededObjectClickededLinkClick = (params: {
```

Comment:
This seems to be a very long variable name. Can't it be shortened?

Reply:
something got messed up here. I've changed it to `trackBookmarkLinkClicked`